api/src/pcapi/connectors/titelive.py

```diff
@@ -199,6 +199,10 @@ class TiteliveBase(enum.Enum):
 
 
 def search_products(titelive_base: TiteliveBase, from_date: datetime.date, page_index: int) -> dict[str, typing.Any]:
+    """Search products in Titelive.
+    returns a product for which an article has been modified since from_date
+    all the articles are returned even if they have not been modified recently.
```

Comment:
```suggestion
    Returns Titelive oeuvres for which an article has been modified since from_date.
    All the articles of an oeuvre are returned even if they have not been modified recently.
```